lib/keys.js

```diff
@@ -129,6 +129,31 @@ exports.pttl = function (mockInstance, key, callback) {
   });
 };
 
+/**
+ * PERSIST
+ * http://redis.io/commands/persist
```

Comment:
PERSIST  should remove the timeout?

Reply:
yep, it should set to `-1`

Reply:
looks like the persist function wont return -1.

can you add tests to clarify?